test/handler/test_ansible_lint_handler.vader

```diff
@@ -1,58 +0,0 @@
-Before:
-  runtime ale_linters/ansible/ansible_lint.vim
-  call ale#test#SetFilename('main.yml')
-
-  let b:ale_warn_about_trailing_whitespace = 1
-
-After:
-  unlet! b:ale_warn_about_trailing_whitespace
-
-  call ale#linter#Reset()
-
-Execute(The ansible-lint handler should handle basic errors):
```

Comment:
Replace these tests with new ones that test the parsing of errors. We ought to be able to still parse these messages.

Reply:
As @w0rp says these tests should not be modified. Instead new ones are to be added and the Handle() method must work for both the old and new error messages.